applications/ecallmgr/src/ecallmgr_fs_notify.erl

```diff
@@ -215,8 +215,8 @@ register_overwrite(JObj, Props) ->
                    ,Realm
                   ),
     SipUri = nksip_unparse:uri(#uri{user=Username, domain=Realm}),
-    PrevBody = wh_util:to_list(<<"Replaced-By:", NewContact/binary>>),
-    NewBody = wh_util:to_list(<<"Overwrote:", PrevContact/binary>>),
+    PrevBody = wh_util:to_list(<<"Replaced-By:", (wh_util:to_binary(NewContact))/binary>>),
```

Comment:
That's what I'm saying though; if `PrevContact` is 'undefined', building `NewBody` would crash, no? I think I just typed it poorly previously.